pkg/sso/client.go

```diff
@@ -87,11 +94,18 @@ func (c *Client) GetAuthorizationURL(opts GetAuthorizationURLOptions) (*url.URL,
 	c.once.Do(c.init)
 
 	query := make(url.Values, 5)
-	query.Set("domain", opts.Domain)
 	query.Set("client_id", c.ProjectID)
 	query.Set("redirect_uri", c.RedirectURI)
 	query.Set("response_type", "code")
 
+	if opts.Domain == "" && opts.Provider == "" {
+		return nil, errors.New("incomplete arguments. Need to specify either a 'domain' or 'provider'")
```

Comment:
see my previous comment on error messages

Reply:
`incomplete arguments: missing domain or provider`

It avoids capital letters and we don't need to suggest what to do since the error message is pretty clear about the issue.

Reply:
Fixed